docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -157,12 +157,14 @@ ifndef ignore-not-found
 endif
 
 .PHONY: install
-install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | $(KUBECTL) apply -f -
+install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config. No-op if none exist.
+	@out="$$( $(KUSTOMIZE) build config/crd 2>/dev/null || true )"; \
+	if [ -n "$$out" ]; then echo "$$out" | $(KUBECTL) apply -f -; else echo "No CRDs to install; skipping."; fi
 
 .PHONY: uninstall
-uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
-	$(KUSTOMIZE) build config/crd | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
```

Comment:
What about we do like

```
.PHONY: uninstall
uninstall: manifests kustomize
	@if [ -d config/crd ] && [ -f config/crd/kustomization.yaml ]; then \
		$(KUBECTL) delete -k config/crd --ignore-not-found=$(ignore-not-found); \
	else \
		echo "No CRDs to delete; skipping."; \
	fi
```

That is clearer.

Reply:
fixed